gym/wrappers/test_flatten_observation.py

```diff
@@ -19,12 +19,12 @@ def test_flatten_observation(env_id):
         space = spaces.Tuple(
             (spaces.Discrete(32), spaces.Discrete(11), spaces.Discrete(2))
         )
-        wrapped_space = spaces.Box(-np.inf, np.inf, [32 + 11 + 2], dtype=np.float32)
+        wrapped_space = spaces.Box(-np.inf, np.inf, [32 + 11 + 2], dtype=np.float64)
     elif env_id == "KellyCoinflip-v0":
         space = spaces.Tuple(
             (spaces.Box(0, 250.0, [1], dtype=np.float32), spaces.Discrete(300 + 1))
         )
-        wrapped_space = spaces.Box(-np.inf, np.inf, [1 + (300 + 1)], dtype=np.float32)
+        wrapped_space = spaces.Box(-np.inf, np.inf, [1 + (300 + 1)], dtype=np.float64)
```

Comment:
```suggestion
        low = np.zeros((302,), dtype=np.float64)
        high = np.array([250.] + [1.] * 301, dtype=np.float64)
        wrapped_space = spaces.Box(low, high, [1 + (300 + 1)], dtype=np.float64)
```

The dtype is correct, but the bounds should be `np.zeros((302,), dtype=np.float64)` and `np.array([250.] + [1.] * 301, dtype=np.float64)`.